benchmark/papr.js

```diff
@@ -5,6 +5,7 @@ import { papr, COLLECTIONS } from './setup.js';
 const sampleSchema = schema(
   {
     age: types.number(),
+    salary: types.number(),
```

Comment:
```suggestion
    salary: types.decimal(),
```

Reply:
Decimal is used here so include an example monetary value in the benchmark and illustrate the change

Reply:
I think @joshuat's suggested change is still required here.

Reply:
Not sure why this change should be done?  It was added specifically for this change with the new decimal type.  See the previous comment.  Let me know if I am misunderstanding.

Reply:
In the Mongoose schema you defined `salary` as a `Decimal128` type. The Papr schema should match that type and use the new type this PR introduces.

Reply:
oh my mistake, changed it back to number during testing and thought it was already decimal- was looking at it backwards.  Updated.